lib/web/scripts/oneoff/oneoff.sh

```diff
@@ -20,17 +21,9 @@ ARCH=$({{.BinUname}} -m)
 trap 'rm -rf -- "$tempDir"' EXIT
 
 teleportTarballName() {
-    if [ "${OS}" = "Darwin" ]; then
-        if [ "$fips" = "true" ]; then
-            echo "FIPS version of Teleport is not compatible with MacOS. Please run this script in a Linux machine."
-            return 1
-        fi
-        echo "${teleportFlavor}-${teleportVersion}-darwin-universal-${packageSuffix}"
-        return 0
-    fi;
-
     if [ "${OS}" != "Linux" ]; then
-        echo "Only MacOS and Linux are supported." >&2
+        echo "ERROR: This script works only for Linux. Please go to the downloads page to find the proper installation method for your operating system:" >&2
+        echo "https://goteleport.com/download/" >&2
```

Comment:
If I understand correctly this breaks macos usage of oneoff.sh. We cannot do that as the script is also used to download `teleport` and run integrations logic locally for users running MacOS.

Reply:
cc @marcoandredinis once the PR is changed, could you check if the discovery flows using onesoff.sh are still working

Reply:
That's correct.
We must continue to support the Darwin OS target.

This script is used for multiple things:
- installing teleport
- bootstrapping AWS IAM resources (Roles, Identity Providers, 
- bootstrapping external audit storage related resources (S3 buckets, athena, ...)

Reply:
@hugoShaka @vapopov Should we use a different template for the cluster-hosted install.sh script? Currently this script attempts to execute `teleport-update` on macOS, instead of outputting a clear error.

Reply:
I'd rather add a SupportedOSes param to the script instead of duplicating it. oneoff.sh is extensively tested and I wish to keep this property.